providers/instance.rb

```diff
@@ -0,0 +1,28 @@
+use_inline_resources
+
+def whyrun_supported?
+  true
+end
+
+action :create do
+  config_dir = node['ganeti-instance-image']['config_dir']
+  directory "#{config_dir}/networks/instances/" do
```

Comment:
Another instance where `File.join` should probably be used.  There are still several; I can replace all those for you if you like.